app/src/main/resources/log4j.xml

```diff
@@ -143,4 +143,4 @@
         <level value="ERROR" />
```

Comment:
So, do I get this right: Everything in this file can be override thru the means of `cas.properties.conf.tpl`?

Reply:
yes - i changed the default values in the log4j to have a uniformed base.